architecture/additional_concepts/authorization.adoc

```diff
@@ -228,17 +228,192 @@ TIP: Remember that xref:authentication.adoc#users-and-groups[users
+The verbs are described in the following table.
+
+.Cluster and Local Policy Verbs
+[cols="3",options="header"]
+|===
+
+|Verb
+|Read/Write Policy
+|Description
+
+|create
+|ReadWrite
+|Allows for new resource object to be created in the storage backend. 
+
+|delete
+|ReadWrite
+|Allows for removal of resource object. Depending on the specific resource, child objects may or may not be garbage collected by the server. See notes on specific resource objects for details.
+
+|deletecollection
+|ReadWrite
+|Allows for resource collection deletion for namespace cleanup.
+
+|get
+|ReadOnly
+|Allows for specific resource object to be retrieved by name.
+
+|impersonate
+|ReadWrite
+|Allows the holder (user, group, or userextra object) to pass requests with the policy bindings of other service accounts in that namespace.
+
+|list
+|ReadOnly
+|Allows for retrieval of all resource objects of a specific type within a namespace.
+
+|patch
+|ReadWrite
+|Allows for changes applied to a specific field in a resource object. How the change is merged is defined per field. Lists may either be replaced or merged; merging lists will not preserve ordering.
+
+|proxy
+|ReadWrite
+|Allows for the creation of proxy server or application-level gateway between localhost and the Kubernetes API Server.
+
+|update
+|ReadWrite
+|Allows for resource object to be replaced with a complete and modified configuration.
+
+|view
+|ReadOnly
+|Allows the holder to retrieve job information from Jenkins builds.
+
+|watch
+|ReadOnly
+|Watch will stream results for a resource object as it is updated. Similar to a callback, watch is used to respond to resource changes.
+
+|*
+|ReadWrite
+|All verbs are allowed for the specified resource object.
+
+|===
+
+The following tables show the verb-resource relationships for common cluster and local policy roles.
+
+.admin Rules Matrix
+[cols="2s,8l"]
+|===
+|create
+|pods pods/attach pods/exec pods/portforward pods/proxy configmaps endpoints persistentvolumeclaims replicationcontrollers replicationcontrollers/scale secrets serviceaccounts services services/proxy horizontalpodautoscalers cronjobs jobs scheduledjobs deployments deployments/rollback deployments/scale horizontalpodautoscalers jobs replicasets replicasets/scale replicationcontrollers/scale statefulsets rolebindings roles localresourceaccessreviews localsubjectaccessreviews subjectrulesreviews podsecuritypolicyreviews podsecuritypolicyselfsubjectreviews podsecuritypolicysubjectreviews buildconfigs buildconfigs/webhooks builds buildconfigs/instantiate buildconfigs/instantiatebinary builds/clone deploymentconfigs deploymentconfigs/scale generatedeploymentconfigs deploymentconfigrollbacks deploymentconfigs/instantiate deploymentconfigs/rollback imagestreamimages imagestreammappings imagestreams imagestreams/secrets imagestreamtags imagestreamimports routes processedtemplates templateconfigs templates buildlogs resourceaccessreviews subjectaccessreviews 
```

Comment:
All these values appear in a single scrollable line when rendered, it will make sense to break all these into separate lines for better readability. 

This applies to all (2nd column) values in this table